ts/packages/node/src/stores/LaunchDarkly.ts

```diff
@@ -0,0 +1,176 @@
+import { Config, ConfigStore, ConfigStoreQuery } from '@configu/ts';
+import axios, { Axios } from 'axios';
+import * as fs from 'fs';
+import _ from 'lodash';
+
+export interface LaunchDarklyConfigStoreParams {
+  apitoken?: string;
+  defaultproject?: string;
+  ldcJsonPath?: string;
+  ldcConfiguration?: string;
+}
+
+export class LaunchDarklyConfigStore extends ConfigStore {
+  private client: Axios;
+  private readonly projectKey: string;
+
+  constructor(configurations: LaunchDarklyConfigStoreParams) {
+    super('launch-darkly');
+    // * ldc.json is the default config file name. see https://github.com/launchdarkly-labs/ldc
+    const fileConfigs =
+      configurations.ldcJsonPath && configurations.ldcConfiguration
+        ? JSON.parse(fs.readFileSync(configurations.ldcJsonPath).toString())[configurations.ldcConfiguration]
+        : {};
+    const configs = {
+      ...{ apitoken: configurations.apitoken, defaultproject: configurations.apitoken },
+      ...fileConfigs,
+    };
```

Comment:
`defaultproject: configurations.apitoken` is definitely a mistake.

I suggest we do this:
```node.js
    const { ldcJsonPath, ldcConfigurationKey, ...restConfigurations } = configurations;
    const ldcConfigurations =
      ldcJsonPath && ldcConfigurationKey
        ? JSON.parse(fs.readFileSync(ldcJsonPath).toString())[ldcConfigurationKey]
        : {};
    const configs = _.merge(restConfigurations, ldcConfigurations);
```

Reply:
definitely. thanks.
i see you've grown in to lodash way of life.